applications/harvester-api/src/main/resources/logback.xml

```diff
@@ -1,31 +1,47 @@
-<?xml version="1.0" encoding="UTF-8"?>
 <configuration>
-    <include resource="org/springframework/boot/logging/logback/base.xml"/>
-    <logger name="org.springframework.web" level="WARN"/>
-    <logger name="org.apache.http.wire" level="WARN"/>
-    <logger name="org.apache.jena" level="ERROR"/>
-	<logger name="org.elasticsearch" level ="WARN"/>
-	<logger name="no.difi.dcat" level ="INFO"/>
-	<logger name="no.dcat" level = "DEBUG"/>
+    <property resource="application.yml" />
```

Comment:
jeg er ikke helt sikker hva den linje gjør.

Reply:
Da er vi to! Men dette ser ut til å virke for api-cat

Reply:
Og som du sier, vi har logging config i application yml som mest sannsynlig er viktig